tmt/plans/integration.fmf

```diff
@@ -3,15 +3,31 @@ provision:
   image: $@{test_disk_image}
 prepare:
   # Replace package mode with image mode
+  # DO NOT RUN ON Github or Locally
```

Comment:
It's a bit unclear what "DO NOT RUN ON Github" means. The term "unify" also isn't obvious to me.

How about calling this "init_package_mode" or something - making clear that in this flow we're starting from package mode.